src/scripts/main.js

```diff
@@ -1,3 +1,19 @@
 'use strict';
 
-// write your code here
+const data = [...document.querySelectorAll('.population')];
+
+const populations = data.map((value) => {
+  const text = value.innerText.trim();
+  const numericText = text.split(',').join('');
+
+  return Number(numericText);
```

Comment:
Consider handling cases where 'numericText' might not convert to a valid number, which could result in NaN values in the 'populations' array.